client/my-sites/people/viewers-list/viewers.jsx

```diff
@@ -96,6 +96,8 @@ class Viewers extends React.PureComponent {
 
 	render() {
 		const listClass = this.state.bulkEditing ? 'bulk-editing' : null;
+		const isSitePrivateOrComingSoon =
+			this.props.site.is_private || get( this.props.site, 'is_coming_soon', false );
```

Comment:
I don't think we need lodash here, we should be able to get away with nullish coalescing:

```suggestion
			this.props.site.is_private || this.props.site.is_coming_soon ?? false;
```